systems/summit_user_guide.rst

```diff
@@ -1738,6 +1738,22 @@ following policies:
     Eligible-to-run jobs have not started and are waiting for resources.
     Running jobs are actually executing.
 
+``killable`` Queue Policy
+""""""""""""""""""""""""""
+
+The ``killable`` queue is a preemptable queue that allows jobs in bins 4 and 5 to request walltimes up to 24 hours. Jobs submitted to the killable queue will be preemptable once the job reaches a minimum runtime limit. Currently the minimum runtime limit matches the maximum runtime of the job’s corresponding batch queue maximum walltime. For example, a job in bin 5 submitted to the killable queue can request a walltime of 24 hours. The job will be preemptable after two hours of run time. Similarly, a job in bin 4 will be preemptable after six hours of run time. Once a job is preempted, the job will be resubmitted by default. To prevent a preempted job from being requeued, the ``BSUB -rn`` flag can be used at submit time.
```

Comment:
> Once a job is preempted, the job will be resubmitted by default.

I assume this means resubmitted to the killable queue. Does it go back in as "new" job and get the minimum 2 hrs and a possible 24, or does it just get to try to finish the remaining time originally requested?

Reply:
can we also add something about how time is accumulated/charged in the killable queue, and how it affects eligible job limits?